src/components/notifications/index.js

```diff
@@ -0,0 +1,3 @@
+import Static from './static';
+
+export default { Static };
```

Comment:
I would rename the component to something like `ContentNotification` and expose it as that.

Our naming pattern has mostly settled on prefixing the more specific types: `TextInput`, `NumberInput`, .. instead of `Input.Text`, `Input.Number` etc.

I would like to eventually have this consistent across ui-kit. So we can use `ContentNotification`. Then we can later have `PageNotification` and so on in case we ever decide to move those into ui-kit.